src/wast-parser.y

```diff
@@ -743,6 +746,23 @@ const_expr :
     instr_list
 ;
 
+/* Exceptions */
+except :
```

Comment:
I agree that this is not clear. Renamed "except" to "exception_field" to clarify it is a (module) "field".